suites/pacific/rgw/tier-2_rgw_regression.yaml

```diff
@@ -805,17 +805,6 @@ tests:
       name: configure client
       polarion-id: CEPH-83573758
 
-  - test:
```

Comment:
IMO, we should keep this testcase,
since known issue tracker is only for one scenario: GetBucketTagging

we still need coverage for other scenarios covered in s3tests.